ui/shortcutdialog/table.go

```diff
@@ -90,6 +103,56 @@ func (shortcutTable *ShortcutTable) SetShortcuts(shortcuts []*shortcuts.Shortcut
 			SetMaxWidth(1)
 		shortcutTable.table.SetCell(row, shortcutCellIndex, eventCell)
 
+		if *shortcutTable.vimMode != vim.Disabled {
+			normalKey := ""
+			insertKey := ""
+			visualKey := ""
+
+			// HACK
+			// Empty tcell EventKey corresponds to Ctrl-Space
+			if shortcut.VimModifier.NormalEvent == nil {
+				normalKey = EventToString(shortcut.Event)
+			} else if shortcut.VimModifier.NormalEvent.Rune() == rune(tcell.KeyCtrlSpace){
+				normalKey = ""
```

Comment:
this case can be negated and do what the `else` case does. The `else` case can then be removed.